src/OpenTelemetry.Exporter.Geneva/ExceptionStackExportMode.cs

```diff
@@ -14,9 +14,17 @@ public enum ExceptionStackExportMode
     Drop,
 
     /// <summary>
-    /// Exception stack traces are exported as strings.
+    /// Exception stack traces are exported as string, using a culture agnostic representation of the results of ToString() implementation of the Exception.
+    /// Though formatted in culture-agnostic way, this is more targetted towards human readability.
+    /// See <see href="https://learn.microsoft.com/dotnet/api/system.exception.stacktrace"/>.
     /// </summary>
     ExportAsString,
 
+    /// <summary>
+    /// Exception stack traces are exported as string, using the StackTrace property of the Exception.
+    /// See <see href="https://learn.microsoft.com/dotnet/api/system.exception.tostring"/>.
+    /// </summary>
+    ExportAsStackTraceString,
```

Comment:
Alternate option is to do this as a privatepreviewfeature, in the connectionstring. But this seems a safe thing to do, and is opt-in too, so I think this is reasonable approach to take.